demo/unhappy_path.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+exec 2>&1
+(set -x; curl -sS http://localhost:8080/screenings) | ./jq
+(set -x; curl -sS http://localhost:8080/screenings/161) | ./jq
+
+for testname in {\
```

Comment:
Może lepiej było wrzucić do jednego katalogu wszystko co tu wymieniłeś i `for file in dir/*`?